src/index.html

```diff
@@ -17,16 +17,30 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="form form_big"
+      action="search"
+      data-qa="big"
+    >
```

Comment:
The class name 'form_big__input' suggests a BEM naming convention, but it should be 'form__input form__input--big' to correctly represent the block and modifier structure.